tests/sequential/prometheus_metrics/prometheus_metrics_test.go

```diff
@@ -642,6 +643,13 @@ func checkBuildInfo(t *testing.T, families map[string]*prommodel.MetricFamily) {
 	}
 
 	latestCommit := getLatestCommit(t)
+
+	// I haven't found a way to inject the actual value via CI yet, so for now just
+	// tolerate our "we build this in CI" dummy string
+	if _, ok := os.LookupEnv("OPENSHIFT_CI"); ok {
+		latestCommit = "dummy-ci-commit-value"
```

Comment:
The test Pod in CI will have the repo checked out locally. You can add some code in the release repo to export the latest commit hash and then read it here. I am not asking for it to be done immediately but just something to consider for the future.

Reply:
I would _love_ to do exactly as you describe, but I think I have the "reverse" of the problem you're describing, i.e. https://github.com/openshift/release/pull/43390#discussion_r1339355574  -- our tests know the info, but our images don't without it being injected ?